internal/controller/service.go

```diff
@@ -1,36 +1,101 @@
 package controller
 
 import (
+	"fmt"
+	"github.com/kintohub/kinto-cli/internal/api"
 	"github.com/kintohub/kinto-cli/internal/utils"
 	"github.com/olekukonko/tablewriter"
 	"os"
 )
 
-func (c *Controller) Services(envId string) {
+func createTable() *tablewriter.Table {
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetRowLine(true)
+	table.SetHeader([]string{
+		"Name",
+		"Service ID",
+	})
+	table.SetHeaderColor(tablewriter.Colors{tablewriter.Bold, tablewriter.FgWhiteColor},
+		tablewriter.Colors{tablewriter.Bold, tablewriter.FgWhiteColor})
+
+	return table
+}
+
+func (c *Controller) Services(envId ...string) {
+
 	utils.StartSpinner()
 	utils.CheckLogin()
-	blocks, err := c.api.GetBlocks(envId)
+	table := createTable()
+
+	var envName []string
+	var envDetails []api.EnvDetails
+
+	if len(envId) != 0 {
+		blocks, err := c.api.GetBlocks(envId[0])
+
+		if err != nil {
+			utils.TerminateWithError(err)
+		}
+
+		if len(blocks) != 0 {
+			for _, block := range blocks {
+				table.Append([]string{
+					block.DisplayName,
+					block.Id,
+				})
+			}
+			utils.StopSpinner()
+			table.Render()
+		} else {
+			utils.WarningMessage("No services/s found!")
+		}
+
+	} else {
+
+		count := 1
+		envs, err := c.api.GetClusterEnvironments()
```

Comment:
comment - we have a lot of error checking and since this is a CLI, we can consider doing GetSomethingOrDie methods.

Reply:
I've handled error checking and self termination for functions that needs to be called by some other functions so you don't need to handle the errors explicitly. Though the API functions you created pass on the errors to be handled by the cli. This could be converted into get somethingordie